tests/reference/cpp-loop1-0a8cf3b.stdout

```diff
@@ -14,28 +14,32 @@ Kokkos::View<T*> from_std_vector(const std::vector<T> &v)
     return r;
 }
 
-void test_factorial_1(int x)
+int test_factorial_1(int x)
 {
+    int _lpython_return_variable;
     int result;
     if (x < 0) {
-        return;
+        _lpython_return_variable = 0;
```

Comment:
This should first assign to `_lpython_return_variable`, which it does. But then it needs to actually return it using `return _lpython_return_variable`.

Reply:
That's now fixed.